### ISSUE TITLE
ESP32 not discoverable under CSR8510 Bluetooth adapter

### ISSUE THREAD

**Issue by xelons** [nkolban/esp32-snippets#1011]
Hi,

I can scan and find ESP32 on my iPhone as well as on some Android devices, however, it is not shown when using a Bluetooth USB adapter (CSR8510). The Bluetooth adapter is a standard module, it can scan all my other BLE enabled devices except for the ESP32.

Does anyone know why this is the case?
My advertising code looks like this:
`BLEAdvertising *pAdvertising = pServer->getAdvertising();

  pAdvertising->addServiceUUID(BLEUUID(SERVICE_UUID));

  pAdvertising->setScanResponse(true);

  pAdvertising->setMinPreferred(0x06);

  pAdvertising->setScanFilter(false,false);  

  pServer->startAdvertising();`

**Comment by chegewara** [comment]
Did you try with nRF connect app on mobile? It may be hardware issue for example, like very weak signal.

**Comment by xelons** [comment]
@chegewara yes I tried the nRF app, does not show there either. I'm starting to suspect that my Bluetooth USB dongle needs a driver update to be able to start discovering. I'm currently downloading the CSR Bluetooth stack driver to replace the standard windows driver and will see how it goes.

**Comment by chegewara** [comment]
If nRF connect app does not see esp32 then its app or hardware issue (esp32).

**Comment by xelons** [comment]
I use the same esp32 on my iPhone and it connects with the app I developed, I can discover it without issue, it is also discoverable on iPad and my Samsung Android phone.

It is not discoverable however on the CSR8510 Bluetooth adapter, please see attached picture:
https://i.imgur.com/9N9U9iA.png

**Comment by xelons** [comment]
It is also not discoverable on macos :
https://i.imgur.com/0PABEkV.png

**Comment by chegewara** [comment]
I still think its hardware issue, maybe very weak signal, i dont see other explanation.
